addons/zeus/functions/fnc_moduleSimulation.sqf

```diff
@@ -0,0 +1,27 @@
+/*
+ * Author: Fisher, SilentSpike
+ * Toggle Simulation on object.
+ *
+ * Arguments:
+ * 0: The module logic <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Public: No
+ */
+
+#include "script_component.hpp"
+
+params ["_logic"];
+
+if !(local _logic) exitWith {};
+
+private _object = attachedTo _logic;
+if (isNull _object) then {
+    [LSTRING(NothingSelected)] call EFUNC(common,displayTextStructured);
+} else {
+    [QEGVAR(common,enableSimulationGlobal), [_object, !(simulationEnabled _object)]] call CBA_fnc_serverEvent;
```

Comment:
The line makes use of this event defined in the `common` component:
https://github.com/acemod/ACE3/blob/master/addons/common/XEH_postInit.sqf#L135